### ISSUE TITLE
Add page tabbing for Installation docs

### ISSUE THREAD

**Issue by machikoyasuda** [reactioncommerce/reaction-docs#689]
Once this is merged into Docusaurus, https://github.com/facebook/Docusaurus/pull/946, upgrade this repo and use it in Installation docs.

Could also be used in the future to separate pre-2.0 and post-2.0 content on 1 page
